api/ws/rpc/failureCodes.js

```diff
@@ -1,5 +1,65 @@
 'use strict';
 
 module.exports = {
```

Comment:
We are assigning to `module.exports` here and [here](https://github.com/LiskHQ/lisk/pull/734/files#diff-f916757f1eba9ea0e2bf3f2ce114ebcdR29) and then replacing the value on line 65?